### ISSUE TITLE
Grid - disable loading overlay

### ISSUE THREAD

**Issue by amcclain** [exhi/hoist-react#401]
The grid flashes a default `loadingOverlay` with the text "Loading..." in a div with a border. See https://www.ag-grid.com/javascript-grid-overlays/.

+ When exactly does it show? While the grid is busy internally processing and preparing to render data? Do we want this at all? Should we offer an option to disable?
+ If we decide to keep this, suggest we remove the default border via something like:
```css
.ag-theme-balham .ag-overlay-loading-center {
  border: none !important;
}
```

**Comment by amcclain** [comment]
Leaning more and more to removing this altogether - if we found a good use case for this, we could certainly bring it back or make it customizable, but for now I think it would be simpler / more appropriate to zap at the Hoist level.

**Comment by lbwexler** [comment]
ATM and I discusssed -- in favor of removing completely.  Let's do so.

**Comment by amcclain** [comment]
Update - looks like the prop we likely want to default is `suppressLoadingOverlay` - https://www.ag-grid.com/javascript-grid-properties/#overlays

Would ask that we test this out as a default option across our sample apps, then validate in a client app or two to ensure we're not missing anything regarding the functionality here.

**Comment by amcclain** [comment]
So, turning the overlay off entirely made it more clear why it exists - the grids looked overly "empty" on first render in a sub-optimal way.  Resolved by Tom with additional CSS work on 3902ecd033a506c4237947f4226b69f6f2ffdc9c.
